src/components/Charts/charts.scss

```diff
@@ -1,4 +1,6 @@
 #chart {
   background-color: var(--componentBackground);
   padding-top: 2rem;
+  border: solid 2px #fff;
+  border-radius: 8px;
```

Comment:
Why are you mixing rem with pixels and why are you choosing rem here?  Did you read up on the rem post I posted previously?  Here's another one:
https://stackoverflow.com/questions/11799236/should-i-use-px-or-rem-value-units-in-my-css